src/CrossValidation.h

```diff
@@ -30,13 +30,22 @@
 #include "FeatureMemoryPool.h"
 #include "ssl.h"
 
+struct cpCnTriple {
```

Comment:
Could you come up with a more descriptive name for this struct?

Reply:
Updated the struct to "candidateCposCfrac" in the most recent commit.  Let me know if "candidateCposCfracTuple" is better (it's a bit long) and I'll update it.